llvm/test/Transforms/LoopVectorize/induction-wrapflags.ll

```diff
@@ -0,0 +1,114 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals none --version 6
+; RUN: opt -p loop-vectorize -force-vector-width=4 -S %s | FileCheck %s
+
+define void @induction_wrapflags(ptr %p, ptr noalias %q) {
```

Comment:
perhaps something like induction_with_multiple_instructions_in_chain more descriptively describes the interesting bit of the test case?